### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by sunilnc** [dani-garcia/vaultwarden#5289] [Q&A] [answered]
Hi all,

I am trying to update my version of Vaultwarden.

I have tried the following commands:

```
docker stop vaultwarden

docker rm vaultwarden

docker run -d --name vaultwarden --restart unless-stopped -v /bw-data:/data -v /etc/ssl/certs:/ssl -e ROCKET_TLS='{certs="/ssl/bitwarden.crt",key="/ssl/bitwarden.key"}' -p 8080:80 vaultwarden/server:latest
```

However, my instance is still running v1.32.5 despite v1.32.6 being the latest.

Any ideas on what I'm doing wrong?

**Comment by BlackDex** [comment, marked as answer]
You forget to run this.
```bash
docker pull vaultwarden/server:latest
```

Run that command before all the others and it should work.

**Reply by sunilnc** [reply]
Thanks once again @BlackDex.
